app/constants/constants.js

```diff
@@ -194,6 +197,28 @@ export const CONNECTION_CONFIG = {
             'type': 'password',
             'description': 'Your data.world read/write token. It can be obtained from https://data.world/settings/advanced'
         }
+    ],
+    [DIALECTS.ATHENA]: [
+        {
+            'label': 'S3 Access Key', 'value': 'accessKey', 'type': 'password'
+        },
+        {
+            'label': 'S3 Secret Access Key', 'value': 'secretKey', 'type': 'password'
+        },
+        {
+            'label': 'AWS Region', 'value': 'region', 'type': 'text'
+        },
+        {
+            'label': 'S3 Bucket', 'value': 's3Outputlocation', 'type': 'text',
+            'description': 'The Athena will put the output results of the query in this location.'
+        },
+        {
+            'label': 'Database', 'value': 'database', 'type': 'text'
+        },
+        {
+            'label': 'Athena QueryTimeout', 'value': 'queryTimeout', 'type': 'number',
+            'description': 'The maximum duration (In Milliseconds) that an SQL query can execute for.'
+        }
```

Comment:
A few suggestions here:
- no need to repeat the connector's name in the fields
- since there is only one timeout, why not call this field just `Timeout`
- reuse wording from previous connectors
```
        {
            'label': 'Timeout', 'value': 'timeout', 'type': 'number',
            'description': 'Number of milliseconds for a query to timeout.'
        }
```

Reply:
Add to the description what the default value for timeout is